packages/components/src/snackbar/SnackbarProvider.js

```diff
@@ -45,7 +45,7 @@ SnackbarProvider.propTypes = {
 };
 
 SnackbarProvider.defaultProps = {
-  timeout: 4500,
+  timeout: 450000,
```

Comment:
Why was this changed?

Reply:
opps, this needs to reverted

```suggestion
  timeout: 4500,
```